.config/.prettierignore

```diff
@@ -18,4 +19,5 @@
 !test-strategy.md
 **/js/_space-test/**
 js/app/test/gradio_cached_examples/**
```

Comment:
BTW this line seems to be no longer valid.
@pngwn Should it be fixed or removed?

Reply:
Yeah i think that is probably safe to remove now.